src/components/EntityInfo/EntityMenu.vue

```diff
@@ -19,50 +19,48 @@
       <span class="font-14 mx-3">{{ getReasonText || 'Unknown Reason' }}</span>
     </span>
 
-    <template v-else>
-      <!-- View and Change Company Information -->
-      <span v-if="isAllowed(AllowableActions.VIEW_CHANGE_COMPANY_INFO)">
-        <v-btn
-          small text color="primary"
-          id="company-information-button"
-          :disabled="hasBlocker"
-          @click="promptChangeCompanyInfo()"
-        >
-          <v-icon medium>mdi-file-document-edit-outline</v-icon>
-          <span class="font-13 ml-1">View and Change Business Information</span>
-        </v-btn>
-
-        <v-tooltip v-if="isPendingDissolution" top content-class="top-tooltip" transition="fade-transition">
-          <template v-slot:activator="{ on }">
-            <v-icon color="orange darken-2" size="24px" class="pr-2" v-on="on">mdi-alert</v-icon>
-          </template>
-          You cannot view or change business information while the business is pending dissolution.
-        </v-tooltip>
-      </span>
-
-      <!-- Dissolve Company -->
-      <span v-if="isAllowed(AllowableActions.DISSOLVE_COMPANY)">
-        <v-tooltip top content-class="top-tooltip" transition="fade-transition">
-          <template v-slot:activator="{ on }">
-            <v-btn
-              small text color="primary"
-              id="dissolution-button"
-              :disabled="hasBlocker"
-              @click="promptDissolve()"
-              v-on="on"
-            >
-              <img src="@/assets/images/Dissolution_Header_Icon.svg" alt="" class="pa-1">
-              <span class="font-13 ml-1">Dissolve this Business</span>
-            </v-btn>
-          </template>
-          Dissolving the business will make this business historical
-          and it will be struck from the corporate registry.
-        </v-tooltip>
-      </span>
-    </template>
+    <!-- View and Change Business Information -->
+    <span v-if="isBusiness && !isHistorical">
```

Comment:
Here I took a different approach: only show the action when it's appropriate but enable it only if allowed.

Ditto for Dissolve Business below.